lib/signs/utilities/audio.ex

```diff
@@ -394,15 +394,29 @@ defmodule Signs.Utilities.Audio do
       sign,
       Enum.map(audios, &Content.Audio.to_params(&1)),
       Enum.map(audios, &Content.Audio.to_tts(&1)),
-      Enum.map(audios, fn audio ->
-        [
-          message_type: to_string(audio.__struct__) |> String.split(".") |> List.last(),
-          message_details: Map.from_struct(audio) |> inspect()
-        ]
-      end)
+      Enum.map(audios, &format_message/1)
     )
   end
 
+  # Format PA Messages separately to ensure backwards compatibility
+  defp format_message(%PaMessages.PaMessage{} = audio) do
+    message_details =
+      audio
+      |> Map.from_struct()
+      |> Map.drop([:sign_ids])
+      |> Enum.map(fn {k, v} -> {k, inspect(v)} end)
+
+    [message_type: to_string(audio.__struct__) |> String.split(".") |> List.last()] ++
```

Comment:
**suggestion:** [`Module.split/1`](https://hexdocs.pm/elixir/Module.html#split/1) will do that for ya

```suggestion
    [message_type: Module.split(audio.__struct__) |> List.last()] ++
```

Reply:
O nice! TIL!